007-disable-usage-stats.md

```diff
@@ -0,0 +1,118 @@
+# Ability to Disable Usage Stats <!-- What do you want to call your `awesome_feature`? -->
+
+- Implementation Owner: @lohanidamodar
+- Start Date: 12-01-2021
+- Target Date: Unknown
+- Appwrite Issue: https://github.com/appwrite/appwrite/issues/547
+
+## Summary
+
+[summary]: #summary
+
+<!-- Brief explanation of the proposed contribution. Write your answer below. -->
+Due to difficulty in deployment of influxdb and telegraf in services like Heroku, in the Appwrite lite version, usage stats has to be disabled. So, want to introduce a environment variable that will allow disabling usage stats completely there by influxdb and telegraf dependency will no longer be required.
```

Comment:
and also the Appwrite usage worker that sends the data to telegraf.

Reply:
updated.